openapi3/openapi3.go

```diff
@@ -87,6 +87,14 @@ func (doc *T) AddServer(server *Server) {
 	doc.Servers = append(doc.Servers, server)
 }
 
+func (doc *T) AddServers(server []*Server) {
```

Comment:
Better yet, why not take a variadic slice instead of a slice?

Reply:
![image](https://github.com/getkin/kin-openapi/assets/17515601/95722f3a-3c48-45e4-9dcf-faa8e9bdd46f)
I see what you mean... [the main reason it was done by pointer is because the existing ](https://gobyexample.com/variadic-functions), will make that change and update this PR

Reply:
Updated the function